src/components/startPage/StartPage.js

```diff
@@ -16,9 +18,35 @@ function StartPage() {
         setFields(newFields);
     }
 
+    async function buttonBack() {
+        const newFields = { ...field };
+        if (newFields.page !== 1) newFields.page = newFields.page - 1;
+        setFields(newFields);
+        const response = await axios.get(`https://api.flickr.com/services/rest?sort=relevance&parse_tags=1&content_type=7&extras=description%2Cowner_name%2Crealname%2Curl_s%2Curl_n%2Curl_w%2Curl_m%2Curl_z%2Curl_c%2Curl_l&per_page=${newFields.perPage}&page=${newFields.page}&lang=en-US&text=${newFields.keyWord}&viewerNSID=&method=flickr.photos.search&csrf=&api_key=bfe6ce4d5f9d4e425f7bf0b9db4fa329&format=json&hermes=1&hermesClient=1&reqId=5541ed04&nojsoncallback=1`)
+        setSearchResult(response.data.photos)
+        console.log(response)
+    }
+
+    async function buttonForward() {
+        const newFields = { ...field };
+        newFields.page = newFields.page + 1;
+        setFields(newFields);
+        const response = await axios.get(`https://api.flickr.com/services/rest?sort=relevance&parse_tags=1&content_type=7&extras=description%2Cowner_name%2Crealname%2Curl_s%2Curl_n%2Curl_w%2Curl_m%2Curl_z%2Curl_c%2Curl_l&per_page=${newFields.perPage}&page=${newFields.page}&lang=en-US&text=${newFields.keyWord}&viewerNSID=&method=flickr.photos.search&csrf=&api_key=bfe6ce4d5f9d4e425f7bf0b9db4fa329&format=json&hermes=1&hermesClient=1&reqId=5541ed04&nojsoncallback=1`)
+        setSearchResult(response.data.photos)
+    }
+
+    async function pageNumbers(e) {
+        const newFields = { ...field };
+        if(e.target.options.selectedIndex === 1) {newFields.perPage = 25}
```

Comment:
not the best solution to use selectedIndex.
it's better to pass param on handler and use it instead of select indexes.

```
async function pageNumbers(e) {
        const newFields = { ...field };
        newFields.perPage = e.target.value
...
```
look at example: https://stackoverflow.com/questions/28868071/onchange-event-using-react-js-for-drop-down